pyas2/migrations/0002_auto_20190531_1954.py

```diff
@@ -0,0 +1,23 @@
+# Generated by Django 2.2.1 on 2019-05-31 19:54
```

Comment:
Create only one migration file for this change please.

Reply:
@chadgates please take care of the above comment and I will merge

Reply:
done